apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/CharSequenceContainsAtLeastAssertionsSpec.kt

```diff
@@ -36,10 +36,10 @@ class CharSequenceContainsAtLeastAssertionsSpec : Spek({
                 }.toThrow<IllegalArgumentException> { messageContains("Iterable without elements are not allowed") }
             }
         }
-        describe("ignoringCase.atLeast(1).elementsOf") {
+        describe("ignoringCase.elementsOf") {
```

Comment:
a misunderstanding I guess. Please don't adjust the existing test but copy it and adjust it then.

Reply:
sorry it was my mistake, I thought I had already copied it

Reply:
no worries